lib/internal/bootstrap/pre_execution.js

```diff
@@ -443,6 +444,15 @@ function initializeFrozenIntrinsics() {
   }
 }
 
+function initializePrimordialsExposition() {
+  if (getOptionValue('--expose-primordials')) {
+    process.emitWarning(
+      'Accessing `primordials` is for internal testing only. Do not use it.',
+      'global.primordials');
+    global.primordials = primordials;
+  }
```

Comment:
can this be exposed in `internal/test/binding.js` instead?

Reply:
The idea is to be able to run an internal Node.js module that uses `primordials` as a global, as suggested in https://github.com/nodejs/node/pull/36872#issuecomment-758212471

Reply:
well i won't block but i'd rather you had to write `globalThis.primordials = require('internal/test/binding').primordials`.

Reply:
I believe that it should also be added to `internal/test/binding.js`.